docs/rag/images/rag-ingestion-and-chat.excalidraw

```diff
@@ -0,0 +1,12599 @@
+{
```

Comment:
Apologies for the dummy question as I am not an Excalidraw user (yet) but is this necessary?  I was thinking embedded scene option meant the images themselves would be editable in excalidraw after the fact and this config wouldn't be necessary.  It makes a mess our of the commit/repo.

Reply:
No user is required. Just open the individual images in Excalidraw.
I removed the Excalidraw related section if it adds confusion and re-added PNGs with scenes (they got missed in some update). Hope this is clear now.
Squashing asap.